### PR TITLE
fix: address generic type warnings

### PR DESCRIPTION
**Pull request by kweinmeister**

### CODE REVIEW

File: appengine-java8/datastore/src/test/java/com/example/appengine/MetadataKindsTest.java [hunk 1]
```diff
@@ -75,7 +75,7 @@ void printLowercaseKinds(DatastoreService ds, PrintWriter writer) {
     // Start with unrestricted kind query
     Query q = new Query(Entities.KIND_METADATA_KIND);
 
-    List<Filter> subFils = new ArrayList();
+    List<Filter> subFils = new ArrayList<Filter>();
```
Comment (sriram-mahavadi) [inline appengine-java8/datastore/src/test/java/com/example/appengine/MetadataKindsTest.java:78]: nitpick. want to use the diamond operator instead?  (applies to multiple occurrences below).

`List<Filter> subFils = new ArrayList<>();`

File: appengine-java8/datastore/src/test/java/com/example/appengine/MetadataPropertiesTest.java [hunk 2]
```diff
@@ -199,6 +199,7 @@ public void propertiesOfKind_returnsProperties() throws Exception {
     assertThat(properties).containsExactly("beta", "charlie", "delta");
   }
 
+  @SuppressWarnings("unchecked")
```
Comment (sriram-mahavadi) [inline appengine-java8/datastore/src/test/java/com/example/appengine/MetadataPropertiesTest.java:202]: Any particular reason as to why we are suppressing the check here? (applies to multiple occurrences below).

File: appengine-java8/datastore/src/test/java/com/example/appengine/MetadataKindsTest.java (hunk 1 above)
Comment (kweinmeister) [reply appengine-java8/datastore/src/test/java/com/example/appengine/MetadataKindsTest.java:78]: Sure, can use that wherever possible.

File: appengine-java8/datastore/src/test/java/com/example/appengine/MetadataPropertiesTest.java (hunk 2 above)
Comment (kweinmeister) [reply appengine-java8/datastore/src/test/java/com/example/appengine/MetadataPropertiesTest.java:202]: There is an existing warning `Type safety: Unchecked cast from Object to Collection<String>`

here: `return (Collection<String>) propInfo.getProperty("property_representation");`

as `propInfo.getProperty()` returns type `Object`. Is there a change you suggest to address this? I can also remove these changes, leaving the warnings.

File: functions/v2/ocr/ocr-translate-text/src/main/java/functions/OcrTranslateApiMessage.java [hunk 3]
```diff
@@ -59,7 +59,8 @@ public String getLang() {
     return lang;
   }
 
-  public static OcrTranslateApiMessage fromPubsubData(byte[] data) {
+  @SuppressWarnings("unchecked")
```
Comment (averikitsch) [inline functions/v2/ocr/ocr-translate-text/src/main/java/functions/OcrTranslateApiMessage.java:62]: Extra spaces before many of the the "@SuppressWarnings" annotations

**Review by averikitsch — CHANGES_REQUESTED** [review]

File: functions/v2/ocr/ocr-translate-text/src/main/java/functions/OcrTranslateApiMessage.java (hunk 3 above)
Comment (kweinmeister) [reply functions/v2/ocr/ocr-translate-text/src/main/java/functions/OcrTranslateApiMessage.java:62]: Good catch!

**Review by Shabirmean — APPROVED** [review]

**Review by averikitsch — APPROVED** [review]

**Pull request merged**